sale_partner_default_route/models/sale_order_line.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2021 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.multi
+    def _prepare_procurement_values(self, group_id=False):
+        """ Prepare specific key for moves or other components that will be created from
+        a stock rule comming from a sale order line. This method could be override in order
+        to add other custom key that could be used in move/po creation.
+        """
+        values = super(SaleOrderLine, self)._prepare_procurement_values(group_id)
+        self.ensure_one()
+        date_planned = self.order_id.confirmation_date \
```

Comment:
Why this part of code here (that update 'values'). IMHO, it should be limited only to default route behaviour.